common/v2/features/Dashboard/components/MembershipPanel.tsx

```diff
@@ -88,7 +88,12 @@ function MembershipPanel({ history }: Props) {
               <SLink onClick={() => history.push(ROUTE_PATHS.MYC_MEMBERSHIP.path)}>
                 {translateRaw('MANAGE_MEMBERSHIP')}
               </SLink>
-              <SButton inverted={true}>{translateRaw('REQUEST_REWARDS')}</SButton>
+              <Link
+                href="https://support.mycrypto.com/general-knowledge/about-mycrypto/membership-information/"
```

Comment:
Can you add this article to `helpArticles.ts`? And then get the URL as something like `getKBHelpArticle(MEMBERSHIP_INFO)`.

Reply:
I think this is a great idea 👍